### DISCUSSION TITLE
[#1040860] Pausing particles

### DISCUSSION THREAD

**Discussion by SIsilicon** [Mojang/minecraft-editor#109] [Feedback]
Since you currently have the ability to pause entities, and plan on being able to pause block processes, would be great to complete the functionality by also being able to pause particle effects too. This can be used for certain cinematic effects.

**Comment by minecraftchloe** [comment]
YES! Agreed!

**Reply by conmaster2112** [reply]
Do you thing pousing block updates will be also possible? For example send is getting updated after place so it falls, then you can't create sand bridge :)
